### PR TITLE
Past project

### PR DESCRIPTION
**Pull request by Ceemos96**
### Past project review 
in this project i : 
-Added linter to the workflow
-created a new branch
-Added html and css from a past project

### CODE REVIEW

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Ceemos96 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] you haven't reviewed your PR, Please make sure to review your Pull Request as in the requirements 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Portfolio webpage</title>
+
```
Comment (Ceemos96) [inline index.html:7]: Please add a metadata tag for the viewport

File: index.html [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    <meta charset="UTF-8">
+    <title>Portfolio webpage</title>
+    
+  <meta http-equiv="refresh" content="30">
+
+</head>
+
+<link rel="stylesheet" href="styles.css">
+<link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+
```
Comment (Ceemos96) [inline index.html:14]: Your link tag should not be between your head and body tags. Kindly move it to within the head tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Suspendisse et<br> justo. Praesent mattis commodo augue.</p>
+            </div>
+
+
+            <div class="container">
+
+                <div class="box">
+
+                    <h2><i class="fas fa-object-ungroup"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
```
Comment (Ceemos96) [inline index.html:50]: This line of code opens with a "pr" tag. There is no such thing as a pr tag. Please fix it. 🙏

File: index.html [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-code"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
```
Comment (Ceemos96) [inline index.html:58]: This line of code opens with a "pr" tag. There is no such thing as a pr tag. Please fix it. 🙏

File: index.html [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-envelope-open-text"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
```
Comment (Ceemos96) [inline index.html:66]: This line of code opens with a "pr" tag. There is no such thing as a pr tag. Please fix it. 🙏

File: index.html [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+
+
+    <!--  Section Two -->
+
+
+
+    <section class="Section-two-section">
+        <div class="section-two">
+            <div class="header">
+                <h1>How can we help you?</h1>
```
Comment (Ceemos96) [inline index.html:40]: Having two h1 tags is not in line wth the best practice. kindly rectify.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+
+            </div>
+
+
+            <div class="container">
+
+                <div class="box">
+
+                    <h2><i class="fas fa-lightbulb"></i> <san></san>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
```
Comment (Ceemos96) [inline index.html:79]: This line of code opens with a "pr" tag. There is no such thing as a pr tag. Please fix it. 🙏

File: index.html [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-cash-register"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
```
Comment (Ceemos96) [inline index.html:87]: This line of code opens with a "pr" tag. There is no such thing as a pr tag. Please fix it. 🙏

File: index.html [hunk 9]
```diff
@@ -0,0 +1,235 @@
...
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-user-tie"></i> spanDesign</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
```
Comment (Ceemos96) [inline index.html:95]: This line of code opens with a "pr" tag. There is no such thing as a pr tag. Please fix it. 🙏

File: index.html [hunk 10]
```diff
@@ -0,0 +1,235 @@
...
+
+        </div>
+    </section>
+
+
+    <section class="section-three-section">
+
+        <div class="section-three">
+            <div class="section-three-head">
+                <h1>Portfolio</h1>
```
Comment (Ceemos96) [inline index.html:110]: Having more than one h1 tags is not in line wth the best practice. kindly rectify

File: index.html [hunk 11]
```diff
@@ -0,0 +1,235 @@
...
+                <p>Senior graphic designer responsible for daily graphic design content for<br> a medium-sized firm. Worked regularly on graphic, layout, and<br> production materials.</p>
+            </div>
+        </div>
+
+        <div class="container-images">
+            <div><img src="images/pexels-photo-270348.jpeg" alt="">
```
Comment (Ceemos96) [inline index.html:116]: It is best practice to add image alt attributes. kindly give the alt attribute a suitable value.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,235 @@
...
+            </div>
+            <div><img src="images/pexels-photo-574071.jpeg" alt=""></div>
+
+            <div><img src="images/pexels-photo-577585.jpeg" alt=""></div>
+
+            <div><img src="images/pexels-photo-943096.jpeg" alt=""></div>
```
Comment (Ceemos96) [inline index.html:122]: It is best practice to add image alt attributes. kindly give the alt attribute a suitable value.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,235 @@
...
+
+        </div>
+        <div class="exlpore">
+
+            <button class="Explore-buuton">
```
Comment (Ceemos96) [inline index.html:127]: Your button element should have a type attribute.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,235 @@
...
+
+
+
+
+
+    <section class="section-three-section">
+
+        <div class="section-three">
+            <div class="section-three-head">
+                <h1>Our Customers</h1>
```
Comment (Ceemos96) [inline index.html:147]: Having more than one h1 tag is not in line wth the best practice. kindly rectify

File: index.html [hunk 15]
```diff
@@ -0,0 +1,235 @@
...
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Suspendisse et<br> justo. Praesent mattis commodo augue.</p>
+            </div>
+        </div>
+
+        <div class="container-images">
+            <div class=""><img src="images/1.jpg" alt=""></div>
+            <div class=""><img src="images/2%20(1).jpg" alt=""></div>
+            <div class=""><img src="images/3.jpg" alt=""></div>
+            <div class=""><img src="images/5.jpg" alt=""></div>
```
Comment (Ceemos96) [inline index.html:156]: It is best practice to add image alt attributes. kindly give the alt attribute a suitable value.

**Review by Ceemos96 — COMMENTED** [review]
Kindly rectify your code as addressed and submit for a re-review.

cheers 🥂

**Review by KayLemba — APPROVED** [review]
# Project Approved 🚀

Hi @Ceemos96  🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights 
- You included an explanation, a "why", for each requested change 🎉 
-  You phrased the comments politely, as suggestions instead of orders 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Kindly include emojis and gifs to make your code review more human-friendly 🙂 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**